redis_cli.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+#
+# All args passed to this script will be passed to "redis-cli .. --rdb -"
+#
+# This script - workaround with redis-cli >= 6.2 bug, when redis-cli tries to fsync /dev/stdout and exit with error
+# see https://github.com/redis/redis/pull/9135
+#
+
+FILENAME=$(mktemp --suffix=redis-cli-stderr)
+redis-cli $@ --rdb - 2>$FILENAME
+exit_code=$?
+cat $FILENAME >&2
+if [[ $exit_code -ne 0 ]] ; then
+	grep "Fail to fsync" $FILENAME | grep -q "Invalid argument" || exit $exit_code
```

Comment:
Let's print stderr file contents if error is unknown. Then we can always delete temporary files.

Reply:
we print stderr always - in previous line "cat $FILENAME >&2"